packages/astro/src/integrations/index.ts

```diff
@@ -221,17 +221,21 @@ export async function runHookConfigDone({
 								logger
 							);
 							for (const [featureName, supported] of Object.entries(validationResult)) {
-								if (!supported) {
+								// If `supported` / `validationResult[featureName]` only allows boolean,
+								// in theory 'asstes' false, doesn't mean that the feature is not supported, but rather that the chosen image service is unsupported
+								// in this case we should not show an error, that the featrue is not supported
+								// if we would refactor the validation to support more than boolean, we could still be able to differentiate between the two cases
+								if (!supported && featureName !== 'assets') {
```

Comment:
As already laid out in the comment, I believe that in the long term, this message should not be based on a boolean. Instead, we should be able to filter whether a feature is unsupported or partially supported.

Reply:
Yeah, the support here is confusing. In my opinion, it would've been better for it to be more about the Node compatibility of the image service vs the specific services.